### DISCUSSION TITLE
Dropdown Change

### DISCUSSION THREAD

**Discussion by WrongCode-01** [primefaces/community#1690] [PrimeNG]
This is an array that will appear in the HTML dropdown
` this.groupby = [
        { label: 'Item Group', value: 'Item Group' },
        { label: 'Item Subgroup', value: 'Item Subgroup' },
        { label: 'Item', value: 'Item' },
        { label: 'Department', value: 'Department' },
        { label: 'Supplier', value: 'Supplier' },
        { label: 'Supplier & Item Group', value: 'Supplier & Item Group' },
        { label: 'No Grouping', value: 'No Grouping' }
    ];`

This is a method where selecting one of the values ​​from the array will remove or display the HTML component
`$('#GroupBy').change(() => {
        const selectedValue = $('#GroupBy').val();
        if (selectedValue === 'Item Group' || selectedValue === 'Item Subgroup' || selectedValue === 'Department' || selectedValue === 'Item') {
            $('#buttonPeriode').removeAttr('hidden', 'hidden');
            $('#button1Tahun').removeAttr('hidden', 'hidden');
            $('#button3Tahun').removeAttr('hidden', 'hidden');
            $('.ItemGroup').removeAttr('hidden', 'hidden');
            $('#ItemGroups').removeAttr('hidden', 'hidden');
            $('.year-picker').attr('hidden', true); 
            $('.PilihData').removeAttr('hidden', 'hidden');
            $('.company-select').removeAttr('hidden', 'hidden');
            $('.Department').attr('hidden', true); 
            $('#departments').attr('hidden', true); 
        } else if (selectedValue === 'Supplier' || selectedValue === 'Supplier and Item Group') {
            $('#buttonPeriode').removeAttr('hidden', 'hidden');
            $('#button1Tahun').removeAttr('hidden', 'hidden');
            $('#button3Tahun').removeAttr('hidden', 'hidden');
            $('.departments').remove();
            $('.PilihData').removeAttr('hidden', 'hidden');
            $('.year-picker').attr('hidden', true); 
            $('.Department').attr('hidden', true);
            $('#ItemGroups').removeAttr('hidden');
            $('.ItemGroup').removeAttr('hidden', true);
            $('.company-select').removeAttr('hidden');
        }
         });`
    
and this is the html part that displays the groupby array data
`<p-dropdown [options]="groupby" optionLabel="label" [filter]="true" [showClear]="true" placeholder="Select GroupBy" [style]="{'width':'100%'}" id="groupby">
      </p-dropdown>`

For showing data to dropdown no problem. example, when i click one data like "Item Group", i want department hidden and other show

this html for department
`<p-multiSelect[options]="optionListDepartmentHR" placeholder="Select Department" id="departments" [style]="{'width':'100%'}"></p-multiSelect>` 

sorry for my english and explain, maybe someone can help me. Thanks

**Comment by cetincakiroglu** [comment]
Hi,

Could you please create a stackblitz example so we can better understand and identify the issue?
